src/mmw/js/src/core/chart.js

```diff
@@ -346,6 +346,8 @@ function renderCompareMultibarChart(chartEl, name, label, colors, stacked, data)
         if (maxValue) {
             yDomainMax = maxValue * 1.2;
         }
+    } else {
+        yDomainMax = precipitation || 1;
```

Comment:
The fallback's to ensure that if precipitation's set to 0, the graph will still range from 0 to 1.